mfes/content-consumption/src/Pages/Details/Details.tsx

```
@@ -0,0 +1,163 @@
+            <img
+              src={'/static/images/cards/salad.jpg'}
+              style={{ width: '100%', borderRadius: '8px' }}
+            />
+            <Typography
+              variant="h6"
+              sx={{ marginTop: '10px', fontWeight: 'bold' }}
+            >
+              Title
+            </Typography>
+            {syllabusList.map((item) => (
+              <ul>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)